vllm/_custom_ops.py

```diff
@@ -320,6 +320,17 @@ def scaled_fp8_quant(
     return output, scale
 
 
+def dynamic_per_token_scaled_fp8_quant(
+        input: torch.Tensor) -> Tuple[torch.Tensor, torch.Tensor]:
```

Comment:
To fit the existing interface this should probably be merged up with the `scaled_fp8_quant`. How do we want to handle this ? 
I believe the `dynamic-per-token` vs `dynamic-per-tensor` model quantization scheme will come from the model itself and we can plumb it down into the `scaled_fp8_quant` function. I'd appreciate any references into will how to plumb this down. Thanks. 
@robertgshaw2-neuralmagic @comaniac @tlrmchlsmth  ?

Reply:
I think this looks good as is. I can handle the plumbing from here in a follow up with some end-to-end model tests

Reply:
Definitely prefer to have a unified API.